cogs/poll.py

```diff
@@ -0,0 +1,131 @@
+import asyncio
+import discord
+from discord.ext import commands
+from cogs.base import BaseCog
+
+
+class Poll(BaseCog):
+
+    @commands.command(pass_context=True)
+    async def poll(self, ctx, question, minutes, *options: str):
+        """ Generates a poll. Consists of a question, the options to choose from and the number of minutes to keep
+        the poll going
+
+        :param ctx: Context of the command message
+        :param question: Question for the poll
+        :param options: List of options to choose from
+        :param minutes: Number of minutes to run the poll for
+        :return: None
+        """
+        if number_of_options_does_not_fall_in_range(options):
+            await ctx.send('Number of options needs to be more than 1 and less than 10. Number of options given: {}.'
```

Comment:
Convert this to use f strings. Also make the poll max and min static variables of the Poll class. That way we can dynamically change the parameters in 1 spot.